src/compas/robots/model/geometry.py

```diff
@@ -3,28 +3,31 @@
 from __future__ import print_function
 
 import compas
+import compas.colors
 import compas.geometry
 from compas.data import Data
 from compas.datastructures import Mesh
 from compas.files.urdf import URDFElement
-from compas.files.urdf import URDFGenericElement
 from compas.geometry import Frame
-from compas.geometry import Shape
-from compas.utilities import hex_to_rgb
+from compas.robots.model.base import ProxyObject
+from compas.robots.model.base import _attr_from_data
+from compas.robots.model.base import _attr_to_data
+from compas.robots.model.base import _parse_floats
 
 __all__ = [
     'Geometry',
-    'Box',
-    'Cylinder',
-    'Sphere',
-    'Capsule',
     'MeshDescriptor',
     'Color',
     'Texture',
     'Material',
-    'Origin'
+    'Origin',
+    'Cylinder',
+    'Box',
+    'Sphere',
+    'Capsule',
 ]
 
+
 # Copied from https://github.com/ubernostrum/webcolors/blob/master/webcolors.py
 HTML4_NAMES_TO_HEX = {
```

Comment:
perhaps this is something to be included in the color class and maps (not now of course)

Reply:
Added it to Color (https://github.com/compas-dev/compas/pull/983/commits/bb86ac9cc1f469d7954817e73162e27c0455bbf8)